cfme/tests/cloud/test_cloud_object_tag_visibility.py

```diff
@@ -0,0 +1,99 @@
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.cloud.availability_zone import AvailabilityZone
+from cfme.cloud.flavor import Flavor
+from cfme.cloud.instance import Instance
+from cfme.cloud.instance.image import Image
+from cfme.cloud.keypairs import KeyPair
+from cfme.cloud.stack import Stack
+from cfme.cloud.provider.ec2 import EC2Provider
+from utils import version
+from utils.blockers import BZ
+from utils.providers import ProviderFilter
+from fixtures.provider import setup_one_or_skip
+
+pytestmark = [test_requirements.tag, pytest.mark.tier(2)]
+
+
+@pytest.fixture(scope='module')
+def a_provider(request):
+    prov_filter = ProviderFilter(classes=[EC2Provider])
```

Comment:
As far as I understand this feature is provider independant. It would be good to run tests against all providers then.